pgjdbc/src/testFixtures/java/org/postgresql/test/TestUtil.java

```diff
@@ -1026,6 +1026,21 @@ public static String queryForString(Connection conn, String sql) throws SQLExcep
     return value;
   }
 
+  /**
+   * Same as queryForString(...) above but with a single string param.
+   */
+  public static String queryForString(Connection conn, String sql, String param) throws SQLException {
+    PreparedStatement stmt = conn.prepareStatement(sql);
+    stmt.setString(1, param);
+    ResultSet rs = stmt.executeQuery();
+    Assert.assertTrue("Query should have returned exactly one row but none was found: " + sql, rs.next());
+    String value = rs.getString(1);
+    Assert.assertFalse("Query should have returned exactly one row but more than one found: " + sql, rs.next());
+    rs.close();
```

Comment:
try-with-resources?

Reply:
Yes that's copy-pasta from it's sibling method without the arg.